FlowCryptUI/Cell Nodes/InboxCellNode.swift

```diff
@@ -34,6 +35,8 @@ public final class InboxCellNode: CellNode {
 
     private let input: Input
 
+    private lazy var avatarNode: ASImageNode = getAvatarImage(text: input.emailText.string)
```

Comment:
When you open `Sent` or `Drafts` folder, all avatars will start with `T` letter because `input.emailText.string` starts with `To: ...`:

<img width="240" alt="Screenshot 2023-08-15 at 13 53 22" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/5ff4a500-6ee1-42b7-972c-fe8ddbfa3725">

Reply:
Nice finding!. Let me check